disnake/message.py

```diff
@@ -1527,6 +1527,12 @@ async def edit(
             those images will be removed if the message's attachments are edited in any way
             (i.e. by setting ``file``/``files``/``attachments``, or adding an embed with local files).
 
+        .. warning::
+
+            This method cannot be used on messages authored by others, with one exception.
+            The ``suppress_embeds`` parameter can be provided for messages authored by others.
+            Changing ``suppress_embeds`` requires :attr:`~.Permissions.manage_messages`.
```

Comment:
```suggestion
            This method cannot be used on messages authored by others, with one exception.
            The ``suppress_embeds`` parameter can be provided in that case,
            requiring :attr:`~.Permissions.manage_messages` permissions.
```
Perhaps something like this?

Reply:
I updated this somewhat in https://github.com/DisnakeDev/disnake/pull/834/commits/16e8d622f47ef749ee2b3a3c0222434a4b259bda

However, I've also realised that the `delete_after` parameter can be provided for messages authored by others, so I'm not sure of the best way to convey that.

Reply:
Should be fine for now.